site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -129,45 +128,45 @@ However, this caused some confusion about the configuration window and GPU featu
 
 ### What's changed in the UI ?
 
-With the initial release of GLvtx, we've had support for running shaders on the host GPU . So, we had
-a dropdown menu called **Shader Emulation** to select either CPU or GPU. The option has now been renamed
-as **Enable Hardware Shader**. When enabled, it makes use of the host GPU, and when disabled, it uses
-the CPU.
+With the initial release of GLvtx, we added support for running shaders on the host GPU. So, we had
+a dropdown menu called **Shader Emulation** to select either CPU or GPU, to run the shaders. The
+option has now been renamed as **Enable Hardware Shader**. When enabled, it makes use of the host
+GPU, and when disabled, it uses the CPU alone.
```

Comment:
Well, this is even less accurate :(
What I'm meaning is, even it's disabled, it uses some of the GPU (though not much)